msmbuilder/featurizer/featurizer.py

```diff
@@ -313,6 +313,104 @@ def partial_transform(self, traj):
         )
         return result
 
+class LandMarkRMSDFeaturizer(Featurizer):
+    """Landmark Featuizer based on RMSD to one or more reference structures.
+
+    This featurizer inputs a trajectory to be analyzed ('traj') and a
+    reference trajectory ('ref') and outputs the kernelized
+    RMSD of each frame of traj with respect to each frame in ref.
+    The output is a numpy array with n_rows = traj.n_frames
+    and n_columns = ref.n_frames. This uses a exponential/gaussian
+    kernel.
+
+    Parameters
+    ----------
+    reference_traj : md.Trajectory
+        The reference conformations to superpose each frame with respect to
+    atom_indices : np.ndarray, shape=(n_atoms,), dtype=int
+        The indices of the atoms to superpose and compute the distances with.
+        If not specified, all atoms are used.
+    sigma: np.float , dtype=0.3nm
```

Comment:
I can fix the text but i was more worried if the implementation was returning the right numbers.